plugins/buildsystem/build.gradle

```
@@ -28,12 +28,16 @@ gradlePlugin {
 repositories {
     mavenCentral()
     google()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }

```

I believe this URL just corresponds to the gradle plugin portal

```suggestion
    gradlePluginPortal()
```